scripts/custom_lint_checks.py

```diff
@@ -143,6 +149,405 @@ def process_module(self, node):
                         'no-break-after-hanging-indent', line=line_num + 1)
 
 
+# This class has been derived from
+# https://github.com/PyCQA/pylint/blob/master/pylint/extensions/docparams.py#L26.
```

Comment:
For the link, change it to the following one: 
https://github.com/PyCQA/pylint/blob/377cc42f9e3116ff97cddd4567d53e9a3e24ebf9/pylint/extensions/docparams.py#L26

This is so that we chain this to a particular version that we are using.